package.json

```diff
@@ -6,15 +6,22 @@
     "main": "dist/index.js",
     "scripts": {
         "start": "npm run build && node dist/index.js",

```

Comment:
do you think it is a good idea to run the sample in the pipeline to verify that the sample works and has no errors?

Reply:
ok just added that step